tests/tests_api/test_group.py

```diff
@@ -77,24 +87,75 @@ def test_public_group_retrieve(api_client):
         "telegram_chat_url": group.telegram_chat_url,
         "coordinators": [
             {
-                "id": coordinator.id,
+                "id": coordinator.pk,
                 "full_name": coordinator.personal_info.full_name,
             }
             for coordinator in group.coordinators.all()
         ],
         "teachers": [
             {
-                "id": teacher.id,
+                "id": teacher.pk,
                 "full_name": teacher.personal_info.full_name,
             }
             for teacher in group.teachers.all()
         ],
         "students": [
             {
-                "id": student.id,
+                "id": student.pk,
                 "full_name": student.personal_info.full_name,
             }
             for student in group.students.all()
         ],
         "is_for_staff_only": group.is_for_staff_only,
     }
+
+
+class TestPublicGroupStart:
+    def test_public_group_start_general_check(self, api_client):
+        group = baker.make(Group, _fill_optional=True, make_m2m=True)
+
+        with transaction.atomic():
```

Comment:
You don't need atomic here, because .save() is called once and it will be already in one transaction

Reply:
Done